core/ir/aarch64/codec.c

```diff
@@ -6809,9 +6809,9 @@ decode_svemem_vec_imm5(uint enc, aarch64_reg_offset element_size, bool is_prefet
     default: ASSERT_NOT_REACHED();
     }
 
-    *opnd = opnd_create_vector_base_disp_aarch64(
-        zn, DR_REG_NULL, get_opnd_size_from_offset(element_size), DR_EXTEND_UXTX, false,
-        imm5, 0, mem_transfer, 0);
+    *opnd = opnd_create_vector_base_disp_aarch64(zn, DR_REG_NULL,
+                                                 get_opnd_size_from_offset(element_size),
+                                                 0, false, imm5, 0, mem_transfer, 0);
```

Comment:
This change looks suspicious: please explain.

Reply:
I don't understand: you reverted an AArch64 commit in this PR?  Then that is a change from your PR (and likely a change we do not want to merge).

In the old days Github would not hide changes from merges: so a reviewer looking at what changed since the last review would see master branch changes if the author did an update mid-review.  If that is what is happening (because it is a personal repo?) then please do not update this PR branch until the review is finished.

Reply:
Looks like github is successfully hiding merge-from-master changes -- so this was all about an explicitly reverted commit; see below about clearly marking that and indicating what the plan is: only temporary to test the CI or actually proposing to permanently revert (in which case it should be a separate PR and should bring in the authors of the being-reverted PR).